test/model/unreads_test.dart

```diff
@@ -469,6 +469,145 @@ void main() {
         }
       }
     });
+
+    group('moves', () {
```

Comment:
Other cases to test here, in order to properly exercise `_popAllInStreamTopic`:
* the moved messages are an (inhabited) proper subset of the existing unreads in the conversation
* the moved messages are a proper superset of the (inhabited) set of existing unreads in the conversation